packages/cosmos-utils/lib/index.ts

```diff
@@ -0,0 +1,42 @@
+import { Block, Transaction, cosmos } from '@liquality/types'
+import { Sha256 } from '@cosmjs/crypto'
+import { toHex, fromBase64 } from '@cosmjs/encoding'
+
+function normalizeBlock(blockResponse: cosmos.BlockResponse, txs: cosmos.Tx[]) {
+  const normalizedBlock: Block<cosmos.Tx> = {
+    number: parseInt(blockResponse.block.header.height),
+    hash: blockResponse.block_id.hash,
+    timestamp: new Date(blockResponse.block.header.time).getTime() / 1000,
+    size: 0,
+    parentHash: blockResponse.block.header.last_block_id.hash,
+    transactions: txs
+  }
+
+  return normalizedBlock
+}
+
+async function normalizeTx(tx: cosmos.Tx, blockHash: string) {
+  const blockHeight = parseInt(tx.height)
+
+  const normalizedTx: Transaction<cosmos.Tx> = {
+    hash: tx.hash,
+    value: 0,
+    blockHash: blockHash,
+    blockNumber: blockHeight,
+    confirmations: 0, // cosmos has instant validity
```

Comment:
Even if that's the case we must calculate the number of confirmations. E.g. 
`confirmations = currentBlockHeight - blockHeight`

Reply:
In Cosmos there is no such a concept as 'confirmation blocks'. It will be updated to match all other packages.